osgp/protocol-adapter-iec60870/osgp-protocol-simulator-iec60870/src/main/java/org/opensmartgridplatform/simulator/protocol/iec60870/domain/profile/DefaultControlledStationAsduFactory.java

```diff
@@ -88,13 +102,24 @@ public ASdu createShortFloatingPointMeasurementAsdu() {
                 new IeTime56(timestamp) } };
     }
 
-    // TODO: remove these setters when the cucumber mock server can read these
-    // values from a property file
+    /*
+     * TODO: remove these setters when the cucumber mock server can read these
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1135)